src/main/kotlin/com/exactpro/th2/conn/amqp/client/AmqpClient.kt

```diff
@@ -36,16 +36,17 @@ typealias Config = Map<String, String>
 @NotThreadSafe
 class AmqpClient(config: Config, val errorReporter: (Throwable) -> Unit) : IClient {
 
+    private val sessionAliasPropertyName = "sessionAlias"
```

Comment:
```suggestion
    private const val SESSION_ALIAS_PROPERTY = "sessionAlias"
```

also suggest to move it into companion as constant